python/GangaLHCb/Lib/Splitters/GaussSplitter.py

```diff
@@ -77,7 +77,11 @@ def split(self, job):
             first = self.firstEventNumber + i * self.eventsPerJob + 1
             opts = 'from Gaudi.Configuration import * \n'
             opts += 'from Configurables import GenInit \n'
-            opts += 'ApplicationMgr().EvtMax = %d\n' % self.eventsPerJob
+            if isinstance(job.application, GaudiExec):
```

Comment:
@egede Unfortunately not. It seems as if for the old code the options are pickled, so subsequent changes to LHCbApp undo any options for it in the options files (eg. ```LHCbApp().DDDBtag```). For GaudiExec though, the ```LHCbApp().EvtMax``` takes precedence over ```ApplicationMgr().EvtMax``` so we need to use LHCbApp for the GaudiExec.

Ok, I can switch it around to test for Gauss application instead.